IaC/bicep/rg-spoke/cluster.bicep

```diff
@@ -224,7 +224,7 @@ module cluster '../CARML/Microsoft.ContainerService/managedClusters/deploy.bicep
       {
         name: 'npsystem'
         count: 3
-        vmSize: 'Standard_DS3_v2'
+        vmSize: 'Standard_D16pls_v5'
```

Comment:
The SKU here was chosen to mirror the SKU used in the Baseline Architecture.

I don't think we should be changing it because of internal subscription limits.

![image](https://user-images.githubusercontent.com/17914476/203854284-fe221d80-2a49-4063-b972-ac4567c882b0.png)

Reply:
Hi @Gordonby, apologies, I pushed the change to my own repo, not realising that it would get included in the pull request. I will use a feature branch in the future :) Reverted. Could you please approve the PR again?